tools/lint-all

```diff
@@ -379,6 +379,7 @@ def build_custom_checkers(by_lang):
              ('zerver/views/user_settings.py', 'return json_error(error or skipped)'),
              # We can't do anything about this.
              ('zerver/views/realm_filters.py', 'return json_error(e.messages[0], data={"errors": dict(e)})'),
+             ('zerver/views/custom_profile_fields.py', 'return json_error(e.messages[0], data={"errors": dict(e)})'),
```

Comment:
really, this is something where we should change the linter to not complain if you pass a variable in.  Let me look into this.